book/src/lint_configuration.md

```diff
@@ -100,7 +100,7 @@ Suppress lints whenever the suggested change would cause breakage for other crat
 ## `msrv`
 The minimum rust version that the project supports
 
-**Default Value:** `None` (`Option<String>`)
+**Default Value:** `Msrv { stack: [] }` (`crate::Msrv`)
```

Comment:
Huh, why is the default value of MSRV an empty stack? 🤔

Reply:
Ah damn I didn't think about that, I think we need to change that in general since other config values also show internal type names which isn't helpful for public documentation

Reply:
Right, I don't think this has to be done in this PR though. So feel free to r=me here now. But if you feel like doing it in this PR, I won't stop you :upside_down_face: